### ISSUE TITLE
Upgrading Fork using winget results in settings lost

### ISSUE THREAD

**Issue by 24mu13** [ForkIssues/TrackerWin#1822]
When upgrading using winget I lost all the settings:
`winget upgrade Fork.Fork`

Basically for some reason winget is uninstalling Fork and re-installing again from scratch.
The file **settings.json** is deleted so & I lost all my previous repos.

See also: [Fork.Fork.installer.yaml](https://github.com/microsoft/winget-pkgs/blob/master/manifests/f/Fork/Fork/1.82.1/Fork.Fork.installer.yaml).

See also: #424, #57

**Comment by andrew-flatters-bliss** [comment]
They are still from the offical CDN:
https://github.com/microsoft/winget-pkgs/blob/master/manifests/f/Fork/Fork/2.8.2/Fork.Fork.installer.yaml

**Comment by ccecvb** [comment]
I'd love to see winget supported, even if it fails to upgrade. 
I do feel a uncomfortable that the winget package is not provided by @DanPristupov . 

It looks OK right now and is probably provided by somebody with good intentions. But in this day and age you must be cautious with which sources you trust.

**Comment by xmedeko** [comment]
Since Fork was not autoupgrading, I have used winget to upgrade it from 2.16.1 to 2.19.0 and all my settings are lost. I had quite a few custom commands there. Very annoying and should be fixed urgently.

See also #1481

**Comment by DanPristupov** [comment]
@xmedeko Use the official installer and everything will work properly.

**Comment by andrew-flatters-bliss** [comment]
> Since Fork was not autoupgrading, I have used winget to upgrade it from 2.16.1 to 2.19.0 and all my settings are lost. I had quite a few custom commands there. Very annoying and should be fixed urgently.
> 
> See also [#1481](https://github.com/fork-dev/TrackerWin/issues/1481)

Yup just got that this morning too as we use RoboPack in company which uses WinGet as a backend for updating things

**Comment by xmedeko** [comment]
@DanPristupov That's very annoying to use installers for all SW I have installed, that's the purpose of winget to simply it. Many companies even require their employees to do automated upgrades by winget. (This is not a case of our company, but some our clients enforce this policy, so our SW must be possible to install by winget reliably.) If you are targeting mid or big size enterprises, this is a must have feature.

**Comment by xmedeko** [comment]
@DanPristupov https://git-fork.com/ site button "Download Fork for Windows" blue button still points to `https://cdn.fork.dev/win/Fork-2.16.1.exe` and the auto update from UI is also staying at 2.16.1 as I have written already.

When I have changed link to `https://cdn.fork.dev/win/Fork-2.19.0.exe` and installed the app, then the installer has deleted my settings, too.

**Comment by DanPristupov** [comment]
@xmedeko the default update channel points to 2.19.0 (https://fork.dev/update/win/RELEASES). If you switched app to stable update channel, you must wait for it.

**Comment by xmedeko** [comment]
@DanPristupov Important is, that even installation from https://cdn.fork.dev/win/Fork-2.19.0.exe nukes settings.

Note: It's confusing that the default channel does not match default version downloadable from web.
